scripts/zones/Southern_San_dOria/npcs/Victoire.lua

```diff
@@ -0,0 +1,58 @@
+-----------------------------------
+-- Area: Southern San d'Oria
+-- NPC: Victoire
+-- Standard Merchant NPC
+-----------------------------------
+package.loaded["scripts/zones/Southern_San_dOria/TextIDs"] = nil;
+-----------------------------------
+
+require("scripts/zones/Southern_San_dOria/TextIDs");
+require("scripts/globals/shop");
+
+-----------------------------------
+-- onTrade Action
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end; 
+
+-----------------------------------
+-- onTrigger Action
+-----------------------------------
+
+function onTrigger(player,npc)
+    
+    player:showText(npc,CARAUTIA_SHOP_DIALOG); -- FIXME: What does Victoire really say?
```

Comment:
I can get you the correct stringID in a couple days (there is a free 10 period coming up, I can spare 5 min to look if nobody else does it before then).